src/components/Lists/index.js

```diff
@@ -44,6 +46,9 @@ function Lists({ results: listsProp, paginationProps, ...props }) {
   useEffect(() => {
     if (data) {
       setLists(data.results);
+      setLoading(true);
+    } else if (!data) {
+      setLoading(false);
```

Comment:
Check both `data` and `error` on results of `useSWR` to be sure if we're in a loading state. [Docs](https://swr.vercel.app/docs/getting-started#make-it-reusable).